src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const TYPE_VALUE_CLEAR = 'clear';
+  const TYPE_VALUE_REMOVE_PROPERTIES = 'removeProperties';
```

Comment:
you tried to do like in redux but these variables are not necessary in this case